backend/src/v3/cabinet/cabinet.controller.ts

```diff
@@ -13,4 +36,167 @@ export class CabinetController {
   constructor(private cabinetService: CabinetService) {}
 
   private logger = new Logger(CabinetController.name);
+
+  @ApiOperation({
+    summary: '사물함 정보 호출',
+    description: 'cabinet_id를 받아 특정 사물함의 상세정보를 받아옵니다.',
+  })
+  @ApiOkResponse({
+    type: CabinetInfoResponseDto,
+    description:
+      '파라미터로 받은 사물함의 정보를 CabinetInfoResponseDto 형식으로 받아옵니다',
+  })
+  @ApiBadRequestResponse({
+    description: '비정상 파라미터',
+  })
+  @Get('/:cabinet_id')
+  async getCabinetInfo(
+    @Param('cabinet_id', ParseIntPipe) cabinet_id: number,
+  ): Promise<CabinetInfoResponseDto> {
+    this.logger.debug(`Called ${this.getCabinetInfo.name}`);
+    return await this.cabinetService.getCabinetResponseInfo(cabinet_id);
+  }
+
+  /**
+   * FIXME: enum validation을 위한 pipe를 추가해야할 것 같습니다.
+   */
+  @ApiOperation({
+    summary: '사물함 상태 변경',
+    description: 'cabinet_id를 받아 사물함의 상태를 변경합니다.',
+  })
+  @ApiNoContentResponse({
+    description: '성공',
+  })
+  @ApiBadRequestResponse({
+    description: '비정상 상태 및 cabinet_id',
+  })
+  @Patch('/status/:cabinet_id/:status')
+  async updateCabinetStatus(
+    @Param('cabinet_id', ParseIntPipe) cabinet_id: number,
+    @Param('status') status: CabinetStatusType,
```

Comment:
```suggestion
    @Param('status', new ParseEnumPipe(CabinetStatusType)) status: CabinetStatusType,
```

이렇게 처리하시면 enum에 등록되지 않은 문자열이 들어올 경우 `Validation failed (enum string is expected)` 로 던져줍니다!